---
pagetitle: "CV"
output: html_document
self_contained: false
---

<style>
.fold-btn { float: left; }
.br2 {
    line-height: 8px;
 }

.column-left{
  float: left;
  width: 60%;
  text-align: left;
}

.column-right{
  float: right;
  width: 40%;
  text-align: left;
}
</style>


<script type="text/javascript">
$(document).ready(function() {
  $(".fold").prepend("<button class=\"fold-btn\">read more</button>");
  $(".fold").children("code").toggle();
  $(".fold-btn").on("click", function() {
    if($(this).text() === "hide") {
      $(this).text("read more");
    } else {
      $(this).text("hide");
    }
    $(this).next("code").toggle("linear");
  })
});
</script>


```{r setup, include=FALSE, echo=FALSE}
knitr::opts_chunk$set(echo = FALSE, 
                      warning = FALSE,
                      tidy.opts=list(width.cutoff=80), 
                      tidy=TRUE,
                      comment=NA)
library(tidyverse)
library(plotly)
library(cowplot)
library(data.table)
library(DT)
library(visNetwork)
library(viridis)
library(DT)
theme_set(theme_cowplot(12))

org_colors <-
  c("Boston University" = "#CC0000",
    "Siemens Diagnostics" = "#009999",
    "Boston Children's Hospital" = "#003087",
    "University of Texas" = "#BF5700",
    "Genentech" = "#005596",
    "Leidos" = "#850F88")

elegen_color <- "#02c796"
```

# {.tabset}

## Education

### University of Texas at Austin
#### PhD Compuational Biology 2020

<details>

<Summary> --- </Summary>

* Leveraged R, Python, and Unix systems to maintain and analyze large-scale multi-omics datasets on cloud-based platforms via statistical- and machine learning-based methods.
* Analyzed & interpreted NGS data in the development of novel protein therapeutics.
* Investigated novel protein interactions in Vibrio cholerae in close collaboration with experimental biologists.
* Led research projects from design to data collection, data evaluation, and publication.

</details>

<br2>

### Boston University
#### BA Biology 2013 (Cum Laude)


## Experience

<div class="column-left">
### Leidos, San Diego, CA
</div>

<div class="column-right">
### 2021 to Present
</div>

#### Senior Scientist Bionformatics/Computational Biology
<details>

<Summary> --- </Summary>

* Work with collaborators to collate and analyze multi-omics data via bespoke pipelines that utilize state of the art methodology.
* Implement machine learning methods to model biologically relevant phenomena. 
* Aid in the planning and pursuit of large scale research efforts.

</details>

<br2>

<div class="column-left">
### Genentech, South San Francisco, CA
</div>

<div class="column-right">
### 2019 to 2020
</div>

#### Programmer Analyst I
<details>

<Summary> --- </Summary>

* Developed an in-house application and analysis pipeline to process, visualize, and explore transposon sequencing data.
* Integrated open source and custom software to analyze, summarize, and interpret bacterial NGS data including long read WGS and microbiome data.
* Worked with bench scientists to facilitate exploratory analysis of microbial datasets through customized interactive R markdown reports.

</details>

<br2>

<div class="column-left">
### Boston Childrenâ€™s Hospital, Boston, MA
</div>

<div class="column-right">
### 2014 to 2016
</div>

#### Research Assistant II & Lab Manager
<details>

<Summary> --- </Summary>


* Studied key metabolic genes involved in pancreatic cancer through cell and mouse-based gene knockdown and overexpression experiments.
* Investigated the role of novel post-translational modification sites on enzymatic function.
* Managed personnel, equipment, reagents, supplies, procurement, and scheduling.

</details>

<br2>

<div class="column-left">
### Siemens, Walpole, MA
</div>

<div class="column-right">
### 2013 to 2014
</div>

#### Laboratory Assistant II
<details>

<Summary> --- </Summary>

* Designed, implemented, and performed quality testing procedures for immunoassays.
* Optimized and maintained standard procedures in compliance with FDA guidelines.

</details>


## Publications

```{r}

citez <- 
  fread("citations.csv"
        )[, Publication :=
            paste0('<a href="',
                   DOI,
                   '" target="_blank">',
                   Publication,
                   '</a>')
        ][, Authors :=
            str_replace_all(
          Authors,
          "((?:DuPai|DuBois), Cory D)",
          "<b>\\1</b>")]

datatable(citez[,c("Publication",
                    "Journal",
                   "Authors",
                    "Year")],
          escape = FALSE,
              rownames = FALSE,
              filter = 'top', 
          extensions = 'Buttons',
              options = list(
                autoWidth = TRUE,
                ordering = TRUE,
                autoWidth = TRUE,
                columnDefs = 
                  list(
                    list(
                      width = '50px', 
                      targets = list(1,3))),
                dom = 'ftB',
                buttons = c('copy', 'csv', 'excel')
                )
              )
```

## Other

- If you're into bacteria, network visualizations, Shiny apps or all of the above, check out my [co-expression network Shiny App](https://corydupai.shinyapps.io/coexpression_networks/?_ga=2.208486294.545398676.1638211342-339371792.1627078524), developed with Tanvi Ingle.
- Connect with me on <a href = "https://www.linkedin.com/in/cory-dupai/" target="_blank">Linkedin </a>.
- Poke around <a href="https://github.com/corydupai" target="_blank">my github</a> for some (admittedly out-of-date) coding examples.
- My dissertation defense is available for your viewing pleasure:

<br>

<iframe width="640" height="360" src="https://www.youtube.com/embed/kLcIJjivgUU" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>